generated/nodejs/test/errorreporting.js

```diff
@@ -115,11 +105,36 @@ function createErrorEvents() {
       report_location: sourceLocation2
     }
   };
-  api.reportErrorEvent(formattedProjectName, event3, function(err, response) {
+
+  // report the three events in parallel
+  async.eachOf([event1, event2, event3], function(event, index, callback) {
+    api.reportErrorEvent(formattedProjectName, event, function(err, response) {
+      if (err) {
+        console.error(err);
+        return;
```

Comment:
you should invoke `callback(err)`, otherwise `async` can't capture the errors correctly.